actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-platform-core/common-util/src/main/java/org/sunbird/common/models/util/JsonKey.java

```diff
@@ -687,6 +687,20 @@
   // this fields are being stored in type column in user_lookup table
   public static final String USER_LOOKUP_FILED_USER_NAME = "username";
   public static final String USER_LOOKUP_FILED_EXTERNAL_ID = "externalid";
-
+  public static final String CONSUMERID = "consumerId";
+  public static final String OBJECTID = "objectId";
+  public static final String CONSUMERTYPE = "consumerType";
+  public static final String OBJECTTYPE = "objectType";
+  public static final String CONSENT_EXPIRY_IN_DAYS = "consent_expiry_in_days";
+  public static final String EXPIRY = "expiry";
+  public static final String CONSENT = "consent";
+  public static final String CONSENTS = "consents";
+  public static final String CONSUMER_ID = "consumer_id";
```

Comment:
an intent on the var name can add value - just a suggestion.

Reply:
Done